src/Symfony/Component/Mime/MimeTypes.php

```diff
@@ -3145,7 +3209,6 @@ public function guessMimeType(string $path): ?string
         'snd' => ['audio/basic'],
         'snf' => ['application/x-font-snf'],
         'so' => ['application/x-sharedlib'],
-        'so.[0-9]*' => ['application/x-sharedlib'],
```

Comment:
Can you check why this was removed?

Reply:
The update script was updated to only put actual extensions in that list and not patterns (as our API does not deal with patterns). My guess is that the file was not properly regenerated when we did that change last time (otherwise it would have been removed at that time)